ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Iterest Rate Swaps
```

Comment:
```suggestion
title: Interest Rate Swaps
```
typo

Reply:
Hello @abcoathup,

I have just added the standard specification

Reply:
Hello @abcoathup,

I have just added the standard specification

Reply:
Specification still includes "In progress..." (as do other sections).

Reply:
Ok @abcoathup,

Could you check the new version ?